src/style.css

```diff
@@ -0,0 +1,35 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Never reset `margins` and `padding` with *. It takes so much resources. Use `*` only for `box-sizing` properties.